.buildkite/windows.ps1

```diff
@@ -0,0 +1,96 @@
+############################################################################
+# Windows build script for Buildkite
+#
+# Requires the Windows 10 VM to be set up with the compiler,
+# and other tools.
+#
+############################################################################
+
+Set-PSDebug -Trace 1
+
+Write-Output "--- Prepare work directory"
+
+# Avoid long paths on Windows
+$env:STACK_ROOT="D:\s"
+$env:STACK_WORK=".w"
+$env:WORK_DIR="D:\w"
+# Override the temp directory to avoid sed escaping issues
+# See https://github.com/haskell/cabal/issues/5386
+$env:TMP="D:\\tmp"
+
+# Store the original checkout directory
+$env:CHECKOUT_PATH=(Get-Item -Path ".\").FullName
+
+# Temp directory needs to exist
+New-Item -ItemType Directory -Force -Path $env:TMP
+
+$env:PATH="$env:PATH;D:\ghc\ghc-8.2.2\bin;D:\stack;$Env:Programfiles\7-Zip;D:\w"
+
+# Clear out work directory, copy source tree there, then cd into it.
+rd -r -fo $env:WORK_DIR
+mkdir $env:WORK_DIR
+copy-item $env:CHECKOUT_PATH\* $env:WORK_DIR -force -recurse
+cd $env:WORK_DIR
+
+# Pre-built rocksdb for Haskell
+git.exe clone https://github.com/facebook/rocksdb.git --branch v4.13.5
+curl.exe -L 'https://s3.eu-central-1.amazonaws.com/ci-static/serokell-rocksdb-haskell-325427fc709183c8fdf777ad5ea09f8d92bf8585.zip' -o D:\Downloads\rocksdb.zip
+7z x D:\Downloads\rocksdb.zip
+
+# CSL-1509: After moving the 'cardano-sl' project itself into a separate folder ('lib/'), the 'cardano-text.exe' executable fails on AppVeyor CI.
+# After some investigation, it was discovered that this was because 'rocksdb.dll' has to be located in this folder as well, or else the test executable doesn't work.
+copy rocksdb.dll node
+copy rocksdb.dll lib
+copy rocksdb.dll wallet-new
+
+############################################################################
+# Prepare stack and build tools
+#
+
+Write-Output "--- Stack setup and install tools"
+
+stack.exe config --system-ghc set system-ghc --global true
```

Comment:
Should we have a comment, why we set the system-ghc?
```
# Make stack use a pached system-ghc. This will allow us to work around the command
# line argument limit on windows by having GHC copy all libraries into a common folder
# prior to linking; and thus reduces the number library search patch flags passed on the
# command line.
#
# WARNING: if you bump the LTS, make sure you have the right (patched) GHC version in path!
```

Reply:
Yes that's good. I added it.